src/mrpro/operators/models/WASABITI.py

```diff
@@ -41,9 +37,8 @@ def __init__(
 
         References
         ----------
-        .. [1] Papageorgakis C, Casagranda S (2023) Fast WASABI post-processing:
-        Access to rapid B0 and B1 correction in clinical routine for CEST MRI. MRM 102(203-2011).
-               https://doi.org/10.1016/j.mri.2023.06.001
+        .. [PAP2023] Papageorgakis C, Casagranda S (2023) Fast WASABI post-processing: Access to rapid B0 and B1
```

Comment:
Never realized that it is listed here. The paper has nothing to do with the WASABITI Model.
Proc. Intl. Soc. Mag. Reson. Med. 31 (2023): 0906 is probably the best reference